package.json

```diff
@@ -87,7 +87,8 @@
     "typesafe-actions": "^4.0.0",
     "uuid": "^3.3.2",
     "validator": "^10.7.0",
-    "whatwg-fetch": "git://github.com/github/fetch#141665b1477bb25ac1427b2cb3e05203453e30cb"
+    "whatwg-fetch": "git://github.com/github/fetch#141665b1477bb25ac1427b2cb3e05203453e30cb",
+    "yarn": "^1.19.1"
```

Comment:
In there any reason you are  adding yarn as a local package?

Reply:
no. it was a mistake. will remove